tests/unit/transformers/pii/test_anonymizer.py

```diff
@@ -102,9 +107,49 @@ def test__function(self):
         result = AnonymizedFaker._function(instance)
 
         # Assert
+        unique_function.assert_not_called()
         function.assert_called_once_with(type='int')
         assert result == 1
 
+    def test__function_enforce_uniqueness_true(self):
+        """Test that ``_function``.
```

Comment:
Add description, or remove the `that`. Do the same on line 77.

Reply:
👍🏻